leo/cli/commands/execute.rs

```diff
@@ -16,28 +16,29 @@
 
 use super::*;
 use clap::Parser;
-// use snarkos_cli::commands::{Developer, Execute as SnarkOSExecute};
-use snarkvm::{cli::Execute as SnarkVMExecute, prelude::Parser as SnarkVMParser};
+use snarkos_cli::commands::{Developer, Execute as SnarkOSExecute};
+use snarkvm::{
+    cli::{helpers::dotenv_private_key, Execute as SnarkVMExecute},
+    prelude::Parser as SnarkVMParser,
+};
 
 /// Build, Prove and Run Leo program with inputs
 #[derive(Parser, Debug)]
 pub struct Execute {
     #[clap(name = "NAME", help = "The name of the function to execute.", default_value = "main")]
     name: String,
-    #[clap(name = "INPUTS", help = "The inputs to the program. If none are provided, the input file is used.")]
+    #[clap(name = "INPUTS", help = "The inputs to the program.")]
     inputs: Vec<String>,
     #[clap(long, help = "Execute the transition on chain", default_value = "false")]
     broadcast: bool,
-    #[clap(long, help = "Custom priority fee in microcredits", default_value = "1000000")]
-    priority_fee: String,
-    #[clap(long, help = "Custom network", default_value = "testnet3")]
-    network: String,
-    #[clap(long, help = "Custom private key")]
-    private_key: Option<String>,
-    #[arg(short, long, help = "The inputs to the program, from a file. Overrides the INPUTS argument.")]
-    file: Option<String>,
+    #[clap(short, long, help = "Execute a different program from the one in the current working directory.")]
```

Comment:
We're using the snarkOS CLI to generate the transaction. Wouldn't that mean that the executed program is always external?

Reply:
What do you think about the alternative `--local` option? If `leo execute` defaulted to the local program and it does not match the external one, then the execution would be invalid. Users would enable `--local` only if they are sure that they want to execute the local one.